crates/sncast/src/starknet_commands/declare.rs

```diff
@@ -45,6 +45,7 @@ pub async fn declare(
     nonce: Option<FieldElement>,
     build_config: BuildConfig,
     wait_config: WaitForTx,
+    is_scarb_quiet: bool,
```

Comment:
This is bad name beasue it implies it hold current scarb state not the argument that should be passed to scarb.

Reply:
changed